packages/plugin-credentials/test/unit/spec/credentials.js

```diff
@@ -110,6 +110,14 @@ describe(`plugin-credentials`, () => {
         spark.credentials.supertoken = supertoken;
         spark.credentials.userTokens.add(apiToken);
         spark.credentials.userTokens.add(kmsToken);
+
+      });
+
+      describe(`when logging out`, () => {
+        it(`rejects getUserToken`, () => {
```

Comment:
`rejects`. no need to indicate getUserToken because you're already in its describe block.